motoman_gp165r_support/package.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>motoman_gp165r_support</name>
+  <version>0.1.0</version>
+  <description>
+    <p>ROS-Industrial support for the Motoman GP165R (and variants).</p>
+    <p>
+      This package contains configuration data, 3D models and launch files
+      for Motoman GP165R series of manipulators.
+    </p>
+    <p>
+      <b>Specifications</b>
+    </p>
+    <ul>
+      <li>GP165R - Default</li>
```

Comment:
From *Table 5-1: Basic Specifications (YR-1-06VR165-A00)* in *186044-1CD*, it seems this support package models the *Not-equipped* version of the 165R.

```suggestion
      <li>GP165R - Not equipped</li>
```

If that's not correct, we should at least mention which column *Table 5-1* was used for the joint limits.

Reply:
@gavanderhoorn That a good point. Maybe I can add gp165r-equiped.xacro and indicate that people can swap it out if they have the equiped version.

Reply:
Hm. We could add another file. It's another variant, just like the HC10 and HC20 variants essentially.

Might be the most user-friendly thing to do. Otherwise users would have to change these limits themselves.

Reply:
Added a new commit adding information about the 'Equipped' version.  Added a gp165R_macro_equipped.xacro but did not doubled up all the launch and test files. Instead a put comments to switch the file reference in the gp165r.xacro.

Reply:
I've gone ahead and made the *equipped* version a proper variant.

That led to some duplication of files, but I believe that's better UX than users having to edit files (which would be impossible if/when we actually release these packages through the buildfarm).